### ISSUE TITLE
sklearn issued warning message for passing attributes to `check_is_fitted`

### ISSUE THREAD

**Issue by ye-yu** [marl/crepe#44]
`sklearn` issued warning message for passing attributes to `check_is_fitted` as follows:

> FutureWarning: Passing attributes to check_is_fitted is deprecated and will be removed in 0.23. The attributes argument is ignored.
>  "argument is ignored.", FutureWarning)

The sklearn version that issued this warning: 0.22

Ways to reproduce: 
1. Use the package version above
2. Run f0 estimation
```python
rate, audio = scipy.io.wavfile.read('/your wav file.wav')
crepe.predict(audio, rate, viterbi=True)
```

This should not affect the prediction output, am I right?

**Comment by jongwook** [comment]
Correct, and thanks for reporting this! `FutureWarning` does not alter the behavior, so you can ignore them for now.

I am not totally sure how `check_is_fitted` is getting called, it's possibly through keras.

I might consider specifying a upper bound to the `sklearn` version to avoid possible errors when using `sklearn` 0.23 in the future.
